test/end-to-end/test_studio_auto_installs.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
Documenting the intent and reason behind the test up here would be good, along with links (PRs, issues) to any regressions it's trying to catch. Any assumptions the script makes about system state, environment, etc. would also be good to document.

If any of the above can be verified / proven by using code, then that would be even better.